docs/apache-airflow-providers-databricks/connections/databricks.rst

```diff
@@ -72,6 +72,7 @@ Extra (optional)
     * ``azure_tenant_id``: ID of the Azure Active Directory tenant
     * ``azure_resource_id``: optional Resource ID of the Azure Databricks workspace (required if Service Principal isn't
       a user inside workspace)
+    * ``azure_ad_endpoint``: optional host name of Azure AD endpoint if you're using special `Azure Cloud (GovCloud, China, Germany) <https://docs.microsoft.com/en-us/graph/deployments#app-registration-and-token-service-root-endpoints>`_.  Should be specified as hostname with protocol.  For example: ``https://login.microsoftonline.de``.
```

Comment:
```suggestion
    * ``azure_ad_endpoint``: optional host name of Azure AD endpoint if you're using special `Azure Cloud (GovCloud, China, Germany) <https://docs.microsoft.com/en-us/graph/deployments#app-registration-and-token-service-root-endpoints>`_. The endpoint must contain a protocol. For example: ``https://login.microsoftonline.de``.
```

Hostname and protocol are distincts part of a URL, and it is technically not correct to say “hostname with protocol”.